src/app/external-cluster-wizard/steps/external-cluster/provider/gke/component.ts

```diff
@@ -160,6 +174,9 @@ export class GKEClusterSettingsComponent
     this.form.valueChanges.pipe(takeUntil(this._unsubscribe)).subscribe(_ => {
       this._updateExternalClusterModel();
       this._updateExternalMachineDeployment();
+      if (this.controlValue(Controls.Zone)?.main) {
+        this._getGKEKubernetesVersions();
```

Comment:
Issues:
When Zone is Removed -> Remove previous version 
![zone-removed](https://user-images.githubusercontent.com/17727069/182348917-42749c3e-470c-458a-93b7-0b827eb197e0.png)